2-mandatory/1-create-functions.js

```diff
@@ -3,16 +3,13 @@ Write a function that:
 - Accepts an array as a parameter.
 - Returns a new array containing the first five elements of the passed array.
 */
-function first5() {
-}
+const first5 = (arr) => arr.slice(0, 5);
 
 /*
 Write a function that:
 - Accepts an array as a parameter.
-- Returns a new array containing the same elements, except sorted.
-*/
-function sortArray() {
-}
+- Returns a new array containing the same elements, except sorted.*/
+const sortArray = (arr) => arr.slice().sort();
```

Comment:
Indeed great point here Mahri! This is called the Spread operator, and it is used to clone arrays to avoid mutating the original array.
You can read up about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) and [here](https://www.programiz.com/javascript/spread-operator).